### PR TITLE
Remove deprecated styles; consolidate color variables

### PR DESCRIPTION
**Pull request by JoeShep**
I'm opening this PR to begin the refactor/cleanup of Rizzo Next after the launch of beta. I've deleted anything marked as deprecated (after checking that they were not in use anywhere), and moved all color variables into one file so we can start renaming/organizing them into a flexible, cohesive unit.

Not looking to merge this just yet. Just wanted to create a branch where we can start the process of getting Rizzo Next ready for 2016.

Comments and discussion very much needed and welcome

### CODE REVIEW

**Comment by jcreamer898** [comment]
:+1:

File: sass/_color_variables.scss [hunk 1]
```diff
@@ -16,9 +16,31 @@ $h2-secondary-color: #dc221a !default;
...
 $linkblue: #288ad6 !default;
 $lpblue: #297cbb !default;
-$nav-primary-type: #bccee3 !default;
 $subduedgray: #dbe6ec !default;
 $tours-blurb-color: #a0a6ad !default;
 $tours-price-background: #dc1b10 !default;
 $tours-title-color: #474f5a !default;
 $tours-type-color: #c4c7cb !default;
+
+$color-warning: #FFC83F;
```
Comment (omgaz) [inline sass/_color_variables.scss]: Consistency is great, maybe stick to lowercase?

File: sass/_mixins.scss [hunk 2]
```diff
@@ -238,7 +227,7 @@
   }
 }
 
-/// @deprecated
+/// @deprecated but still in use in _navigation.scss
```
Comment (omgaz) [inline sass/_mixins.scss]: :+1:

**Comment by jcreamer898** [comment]
@thomasthesecond can you perhaps have a look at this and see if it's something we should get merged in and if not close it out?

**Pull request merged**